docs/epub33/index.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
+	<head>
+		<meta charset="utf-8" />
+		<title>EPUB 3.3 Structural Tests</title>
+		<script src="https://www.w3.org/Tools/respec/respec-w3c" class="remove"></script>
+		<link rel="stylesheet" href="css/tables.css">
+		<style>
+			/* Table zebra style... */
+
+			table.zebra {
+				font-size: inherit;
+				font: 100%;
+				margin: 1em;
+				width: 100% !important;
+			}
+
+			table.zebra td {
+				padding-left: 0.3em;
+			}
+
+			table.zebra th {
+				font-weight: bold;
+				text-align: center;
+				font-size: 100%;
+				background: hsl(180, 30%, 50%);
+				color: #fff;
+			}
+
+			table.zebra th a:link {
+				color: #fff;
+			}
+
+			table.zebra th a:visited {
+				color: #aaa;
+			}
+
+			table.zebra tr:nth-child(even) {
+				background-color: hsl(180, 30%, 93%)
+			}
+
+			table.zebra th {
+				border-bottom: 1px solid #bbb;
+				padding: .2em 1em;
+			}
+
+			table.zebra td {
+				border-bottom: 1px solid #ddd;
+				padding: .2em 1em;
+			}
+
+			td a {
+				text-decoration: none !important;
+			}
+
+			td summary {
+				font-weight: normal;
+			}
+
+			td.pass {
+				background-color: green;
+			}
+
+			td.fail {
+				background-color: red;
+			}
+
+			col.col_id {
+				width: 25%;
+			}
+
+			col.col_descr {
+				width: 5%;
+				text-align: center;
+			}
+
+			code {
+				color: red;
+			}
+		</style>
+		<script class="remove">
+			var respecConfig = {
+				wgPublicList: "public-epub3",
```

Comment:
I can't comment on the line in the revised file, but this was the epub 3 CG list. It should have been public-epub-wg for the last revision and public-pm-wg now. Not sure if it's used in the base profile, but still worth fixing.

Reply:
Right. I have changed it.

I realize that I had to do a bunch of changes in direction to 34 for when we will indeed use this script at site. I must admit I do not really remember how all this worked; this was the script that was used to combine the output of the epubcheck results with the code itself. It is still down the line when the script must be used; I hope it will not have bitrotted too much...